arangod/Pregel/Graph.h

```diff
@@ -68,54 +67,60 @@ class Edge {
   template <typename V, typename E2>
   friend class GraphStore;
 
+  static_assert(sizeof(std::string) > 2, "");
+  char* _toKey;             // uint64_t
+  size_t _toKeyLength;      // uint64_t
```

Comment:
Can we make `_toKeyLength` a uint16_t and move it further down? This would make the struct more compact and we could save some memory.